tideways/provision.sh

```diff
@@ -75,15 +76,31 @@ function install_xhgui_frontend() {
         cd xhgui
         echo " * Installing xhgui"
         sudo php install.php
-        cp -f "${DIR}/config.php" "/srv/www/default/xhgui/config/config.php"
-        echo " * Restarting MongoDB"
-        service mongod restart
     else
         echo -e " * Updating xhgui..."
         cd /srv/www/default/xhgui
         git pull --rebase origin master > /dev/null 2>&1
         noroot composer update --prefer-dist > /dev/null 2>&1
     fi
+    cp -f "${DIR}/config.php" "/srv/www/default/xhgui/config/config.php"
+    if [[ ! -d "/srv/www/default/php-profiler" ]]; then
+        echo -e " * Installing php-profiler for Xhgui"
+        for version in "7.0" "7.1" "7.2" "7.3" "7.4"
+        do
+            if [[ $(command -v php$version) ]]; then
+                apt install "php${version}-sqlite3" -y &>/dev/null
+            fi
+        done
```

Comment:
would it make more sense to include this in the PHP provisioners instead?

Reply:
Well is required only for tideways, we can avoid to install a package for all the other users.

Reply:
1. Lets have a single command executed rather than multiple for each PHP versions. I'd rather a single command for 5 packages than 5 commands for 1 package
2. There's a lot of flags for `apt-get` used in other provisioners that aren't here, they should be used

Reply:
Also, if PHP profiler is present but PHP SQlite failed, then it will not attempt to fix that scenario, and the provisioner will be stuck in a broken state. The PHP SQLite extension should not be inside this check

Also, the extension is unnecessary if we change from the PDO provider

Reply:
bash code for grouping apt install command:

```sh
        declare -a packages=()
        for version in "7.0" "7.1" "7.2" "7.3" "7.4"; do
            if [[ $(command -v php$version) ]]; then
                packages+=("php${version}-sqlite3")
            fi
        done
        test "${#packages[*]}" -ge 0 && apt install -y "${packages[@]}" &>/dev/null
```

if your code runs with `set -e` you may need to write the last line as


```
        test "${#packages[*]}" -eq 0 || apt install -y "${packages[@]}" &>/dev/null
```

which I find a bit less readable

you can author me with `git commit --author` onto that commit ;)

Reply:
Done, it is better indeed. I need to work more on my Bash skills :-)